source/manual/terraform-cloud.html.md

```diff
@@ -0,0 +1,24 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: Terraform Cloud
+section: Infrastructure
+type: learn
+layout: manual_layout
+parent: "/manual.html"
+---
+
+GOV.UK uses Terraform Cloud for [DNS rollouts](/manual/dns.html) and [managing engineer access to third party services](https://github.com/alphagov/govuk-user-reviewer/). It will also be used for the modules in [govuk-infrastructure](https://github.com/alphagov/govuk-infrastructure) soon.
+
+## Getting Terraform Cloud access
+
+You can sign up for Terraform Cloud and join the GOV.UK organisation yourself. You will need to be a member of the [Technical 2nd Line Support](https://groups.google.com/a/digital.cabinet-office.gov.uk/g/2nd-line-support) Google Group, or the [GOV.UK Terraform Cloud Access](https://groups.google.com/a/digital.cabinet-office.gov.uk/g/GOV.UK_Terraform_Cloud_Access/members) group.
+
+1. Follow this link to [join the GOV.UK Terraform Cloud organisation](https://accounts.google.com/o/saml2/initsso?idpid=C01ppujwc&spid=738388265440&forceauthn=false). By connecting your digital.cabinet-office.gov.uk Google account, you'll be granted access to the GOV.UK organisation.
+2. After connecting Google, you'll be asked to create a Terraform Cloud account (or sign in if you already have one). You'll need to set an account password – which might feel weird because you've already linked Google, but it turns out that's only used to grant access to the GOV.UK organisation, not for SSO login functionality.
```

Comment:
Don't think it's correct to say "not for SSO login functionality" - we are using SSO and you do login with your Google Workspace account (i.e. single set of credentials). 

See this comment: https://github.com/alphagov/govuk-developer-docs/pull/4026#discussion_r1219410641

Reply:
I've tweaked the wording. 👍